app/src/layouts/appLayout/appSidebar/appSidebar.jsx

```diff
@@ -63,38 +79,69 @@ export class AppSidebar extends Component {
     this.props.tracking.trackEvent(eventInfo);
   };
 
-  createTopSidebarItems = () => [
-    {
-      onClick: () => this.onClickButton(SIDEBAR_EVENTS.CLICK_DASHBOARD_BTN),
-      link: { type: PROJECT_DASHBOARD_PAGE, payload: { projectId: this.props.activeProject } },
-      icon: DashboardIcon,
-      message: <FormattedMessage id={'Sidebar.dashboardsBtn'} defaultMessage={'Dashboard'} />,
-    },
-    {
-      onClick: this.props.onClickNavBtn,
-      link: {
-        type: LAUNCHES_PAGE,
-        payload: { projectId: this.props.activeProject },
+  createTopSidebarItems = () => {
+    const { projectRole, accountRole, activeProject, onClickNavBtn } = this.props;
+
+    const topItems = [
+      {
+        onClick: () => this.onClickButton(SIDEBAR_EVENTS.CLICK_DASHBOARD_BTN),
+        link: { type: PROJECT_DASHBOARD_PAGE, payload: { projectId: activeProject } },
+        icon: DashboardIcon,
+        message: <FormattedMessage id={'Sidebar.dashboardsBtn'} defaultMessage={'Dashboards'} />,
       },
-      icon: LaunchesIcon,
-      message: <FormattedMessage id={'Sidebar.launchesBtn'} defaultMessage={'Launches'} />,
-    },
-    {
-      onClick: () => this.onClickButton(SIDEBAR_EVENTS.CLICK_FILTERS_BTN),
-      link: { type: PROJECT_FILTERS_PAGE, payload: { projectId: this.props.activeProject } },
-      icon: FiltersIcon,
-      message: <FormattedMessage id={'Sidebar.filtersBtn'} defaultMessage={'Filters'} />,
-    },
-    {
-      onClick: () => this.onClickButton(SIDEBAR_EVENTS.CLICK_DEBUG_BTN),
+      {
+        onClick: onClickNavBtn,
+        link: {
+          type: LAUNCHES_PAGE,
+          payload: { projectId: activeProject },
+        },
+        icon: LaunchesIcon,
+        message: <FormattedMessage id={'Sidebar.launchesBtn'} defaultMessage={'Launches'} />,
+      },
+      {
+        onClick: () => this.onClickButton(SIDEBAR_EVENTS.CLICK_FILTERS_BTN),
+        link: { type: PROJECT_FILTERS_PAGE, payload: { projectId: activeProject } },
+        icon: FiltersIcon,
+        message: <FormattedMessage id={'Sidebar.filtersBtn'} defaultMessage={'Filters'} />,
+      },
+    ];
+
+    if (projectRole !== CUSTOMER) {
+      topItems.push({
```

Comment:
We should pop this item in case of projectRole === CUSTOMER. Because now, if I change the role from PM to CUSTOMER (f.e.) I will be able to see debug button

Reply:
Decided to leave it as is.